docs/UserGuide.md

```diff
@@ -91,20 +89,63 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 q/true`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 q/false t/criminal`
 
-### Listing: `list`
+#### Adding a visit: 
+
+Adds a visit by the personId, location of visit and date of visit
+
+Format: `add personId location date`
```

Comment:
Shouldn't it be "locationId" instead of "location"?

Reply:
Please refer to it as index rather than id. Reminder that ID is a concept that is hidden from users.